api/src/Messages/ApiResource/MessageUnread.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
```

Comment:
```suggestion
<?php

declare(strict_types=1);
```

Don't forget to add it in all of you files